docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc

```diff
@@ -54,9 +66,10 @@ From this folder, run the following command to generate the converted files:
 ----
 find ./ -name "*.md" \
        -type f | xargs -I @@ \
-       bash -c 'kramdoc \
-               --format=GFM \
-               --output=@@.adoc @@';
+       bash -c '{ echo Converting "@@" && \
+         kramdoc --format=GFM --output=@@.adoc @@ && \
+         echo Done \
```

Comment:
⚠️ `done echo` is probably too much. Let's check what is happening on error

Reply:
@alachambre @passga do we keep the echo or is it extra?
otherwise, we can merge this PR